django/contrib/admin/templates/admin/change_form.html

```diff
@@ -73,15 +73,39 @@
             $(document).ready(function() {
                 $('.add-another').click(function(e) {
                     e.preventDefault();
-                    showAddAnotherPopup(this);
+                    var event = $.Event('django:add-another-related');
+                    $(this).trigger(event);
+                    if (!event.isDefaultPrevented())
+                        showAddAnotherPopup(this);
                 });
                 $('.related-lookup').click(function(e) {
                     e.preventDefault();
-                    showRelatedObjectLookupPopup(this);
+                    var event = $.Event('django:lookup-related');
+                    $(this).trigger(event);
+                    if (!event.isDefaultPrevented())
+                        showRelatedObjectLookupPopup(this);
                 });
+                $('body').on('click', '.related-widget-wrapper-link', function(e) {
```

Comment:
Okay, this looks fine then. It was called `.live()` last time I worked with jQuery (I see that's a deprecated alias  now).